src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilation.cs

```diff
@@ -152,23 +151,32 @@ private unsafe void CompileSingleMethod(CorInfoImpl corInfo, LLVMMethodCodeNode
                     return;
                 }
 
+                
                 if (methodIL.GetExceptionRegions().Length == 0 && !_disableRyuJit)
                 {
-                    var mangledName = NodeFactory.NameMangler.GetMangledMethodName(method).ToString();
-                    var sig = method.Signature;
-                    corInfo.RegisterLlvmCallbacks((IntPtr)Unsafe.AsPointer(ref corInfo), _outputFile,
-                        Module.Target,
-                        Module.DataLayout);
-                    corInfo.InitialiseDebugInfo(method, GetMethodIL(method));
-                    corInfo.CompileMethod(methodCodeNodeNeedingCode);
-                    methodCodeNodeNeedingCode.CompilationCompleted = true;
-                    // TODO: delete this external function when old module is gone
-                    LLVMValueRef externFunc = ILImporter.GetOrCreateLLVMFunction(Module, mangledName, GetLLVMSignatureForMethod(sig, method.RequiresInstArg()));
-                    externFunc.Linkage = LLVMLinkage.LLVMExternalLinkage;
-
-                    ILImporter.GenerateRuntimeExportThunk(this, method, externFunc);
-
-                    ryuJitMethodCount++;
+                    if (GetMethodIL(method).GetExceptionRegions().Length == 0)
+                    {
+                        var mangledName = NodeFactory.NameMangler.GetMangledMethodName(method).ToString();
+                        if (mangledName == "S_P_CoreLib_System_DateTimeOffset___cctor")
```

Comment:
Presumably these changes were debug-only and can be reverted now.

Reply:
Yes, thanks for spotting.